pkg/kn/commands/flags/sink.go

```diff
@@ -147,3 +148,18 @@ func SinkToString(sink duckv1.Destination) string {
 	}
 	return ""
 }
+
+// WriteSink writes a sink to a PrefixWriter which is used in describe command
+func WriteSink(dw printers.PrefixWriter, sink *duckv1.Destination) {
+	subWriter := dw.WriteAttribute("Sink", "")
+	ref := sink.Ref
+	if ref != nil {
+		subWriter.WriteAttribute("Name", sink.Ref.Name)
+		subWriter.WriteAttribute("Namespace", sink.Ref.Namespace)
+		subWriter.WriteAttribute("Resource", fmt.Sprintf("%s (%s)", sink.Ref.Kind, sink.Ref.APIVersion))
+	}
+	uri := sink.URI
+	if uri != nil {
+		subWriter.WriteAttribute("URI", uri.String())
+	}
+}
```

Comment:
while doing the subscription CRUD, separated this util here https://github.com/knative/client/blob/abb75e4da2c45534593bb2684dc3253eef3a7d91/lib/printing/describe.go#L28

lets refer this util at following places

Reply:
Thank you for the information. I updated. Please review again.